components/Team/TeamWorkList.tsx

```diff
@@ -1,42 +1,123 @@
 import { observer } from 'mobx-react';
-import { Ratio, Image, Accordion } from 'react-bootstrap';
+
+import Link from 'next/link';
+import {
+  Ratio,
+  Image,
+  Row,
+  Col,
+  Card,
+  Accordion,
+  Button,
+} from 'react-bootstrap';
 
 import { ScrollListProps, ScrollList } from '../ScrollList';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCalendarDay } from '@fortawesome/free-solid-svg-icons';
+
 import { TeamWork, TeamWorkType } from '../../models/Team';
 import activityStore from '../../models/Activity';
 
 export interface TeamWorkListProps extends ScrollListProps<TeamWork> {
   activity: string;
   team: string;
+  size?: 'sm' | 'lg';
+  onDelete?: (id: any) => void;
 }
 
 @observer
 export class TeamWorkList extends ScrollList<TeamWorkListProps> {
   store = activityStore.teamOf(this.props.activity).workOf(this.props.team);
-
-  static Layout = ({ value = [] }: TeamWorkListProps) => (
+  extraProps: Partial<TeamWorkListProps> = {
+    onDelete: (id: any) => {
+      if (!id) return;
+      this.store.deleteOne(id);
+    },
+  };
```

Comment:
```suggestion

  extraProps: Partial<TeamWorkListProps> = {
    onDelete: id => id && confirm('确定删除该作品？') && this.store.deleteOne(id),
  };

```
1. 整个对象声明了类型，每项字段就能自动推导
2. 类成员间空一行